modules/reactor/src/meshgenerators/FlexiblePatternGenerator.C

```diff
@@ -203,7 +207,14 @@ FlexiblePatternGenerator::FlexiblePatternGenerator(const InputParameters & param
                                    ? getParam<SubdomainName>("background_subdomain_name")
                                    : SubdomainName()),
     _delete_default_external_boundary_from_inputs(
-        getParam<bool>("delete_default_external_boundary_from_inputs"))
+        getParam<bool>("delete_default_external_boundary_from_inputs")),
+    _external_boundary_id(isParamValid("external_boundary_id")
+                              ? getParam<boundary_id_type>("external_boundary_id")
+                              : (boundary_id_type)OUTER_SIDESET_ID),
+    _external_boundary_name(isParamValid("external_boundary_name")
+                                ? getParam<BoundaryName>("external_boundary_name")
+                                : "")
```

Comment:
```suggestion
    _external_boundary_name(isParamValid("external_boundary_name")
                                ? getParam<BoundaryName>("external_boundary_name")
                                : BoundaryName())
```
Set the default here instead.